package.json

```diff
@@ -56,9 +54,8 @@
     "react-router-redux": "^4.0.8",
     "reactstrap": "^8.0.0",
     "redux": "^4.0.1",
-    "redux-mock-store": "^1.5.3",
     "redux-thunk": "^2.3.0",
-    "sinon": "^7.3.2",
+    "regenerator-runtime": "^0.13.2",
```

Comment:
@chekwas88 Please what does this package do? If it is for async function. The project has `@babel/plugin-transform-runtime`